digitalocean_plugin/security.py

```diff
@@ -14,21 +14,138 @@
 # limitations under the License.
 
 import os
+import random
+
+import requests
 
 from cloudify.exceptions import NonRecoverableError
 
 
-def load_token(**_):
-    """
-    This will load a security token from a local file called token.txt
-    A token can be obtained from DigitalOcean by Registering a New Developer or Authorized Application.
-    :return: the security token, as a string
-    :raises: NonRecoverableError if token is not present
-    """
-    cwd = os.path.dirname(__file__)
-    token_path = os.path.join(cwd, 'token.txt')
-    if not os.path.isfile(token_path):
-        raise NonRecoverableError('Missing security token file "%s".' % token_path)
-    with open(token_path, 'r') as f:
-        return f.read()
+class DigitalOceanSecurity(object):
+
+    pubkey_stub = "Cloudify Key {0}"
+    api_endpoint = 'https://api.digitalocean.com/v2/'
```

Comment:
If this is reused in other files, you might consider creating a 'constants.py' and declaring the constant there.
See: https://github.com/cloudify-cosmo/cloudify-aws-plugin/blob/master/ec2/constants.py#L53